eclass/emboss.eclass

```diff
@@ -120,16 +128,17 @@ emboss_src_prepare() {
 emboss_src_configure() {
 	econf \
```

Comment:
when using the autotoole-utils eclass we should use myeconfargs=() + autotools-utils_src_configure()